packages/ignite-cli/src/commands/plugin.js

```diff
@@ -85,11 +85,13 @@ const createNewPlugin = async (context) => {
     (answers.template === 'Yes') &&
       { template: 'plugin/templates/Example.js.ejs', target: `${pluginName}/templates/${name}Example.js.ejs` },
     (answers.command === 'Yes') &&
-      { template: 'plugin/commands/example.js.ejs', target: `${pluginName}/commands/example.js` }
+      { template: 'plugin/commands/thing.js.ejs', target: `${pluginName}/commands/thing.js` },
+    (answers.command === 'Yes') &&
+      { template: 'plugin/templates/thing.js.ejs.ejs', target: `${pluginName}/templates/thing.js.ejs` }
```

Comment:
If we have a generator command, we also need a template from which to generate. This is referring to that.